aubreylib/system.py

```diff
@@ -1,9 +1,6 @@
-import urlparse
 import os
 import re
-import urllib
-import urllib2
-
+import urllib.request
```

Comment:
What about `urllib.parse`?

Reply:
:/ seems like that should have failed some test? maybe we need to add more tests...

Reply:
it doesn't even fail when I import that module... not sure why

Reply:
oh right, gotta use one of the classes/functions :D

Reply:
but yeah, it's odd that no tests complained

Reply:
To note, for some reason this works, but let's import explicitly for clarity:
```
Python 3.7.3 (default, Jun 27 2019, 16:48:23) 
[GCC 7.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import urllib.request
>>> urllib.parse.urlsplit('http://unt.edu/something/one/file.txt')
SplitResult(scheme='http', netloc='unt.edu', path='/something/one/file.txt', query='', fragment='')
```
```
Python 3.7.3 (default, Jun 27 2019, 16:48:23) 
[GCC 7.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> urllib.parse.urlsplit('http://unt.edu/something/one/file.txt')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'urllib' is not defined
```

Reply:
Hmm..I was looking at the tests if they were covering the file locations with http and https and they are. It seems weird the way `urlsplit` worked without importing `urllib.parse` and just with `urllib.request` but I'll add it to the imports as you suggested.